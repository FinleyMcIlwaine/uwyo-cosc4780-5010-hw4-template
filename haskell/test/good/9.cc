// Macros to shut up the C++ warnings
#define printInt(x)
#define printDouble(x)
#define printString(x)
#define readInt() 0
#define readDouble() 0
#define readString() 0

#include <iostream>
#include <string>

using std::string;

double shadowMePlz(int x, bool y)
{
    if (y)
    {
        double x = 100.202;
        return x;
    }
    else
    {
        return 6.2352;
    }
    return 4.252;
}

int main()
{
    int x = 10;
    int y = 5;
    double dub = shadowMePlz(x = 2,y >= x);
    printInt(x + y);
    printDouble(dub);
    return 7 - (x + y);
}
