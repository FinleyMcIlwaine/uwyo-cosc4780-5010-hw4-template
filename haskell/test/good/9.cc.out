7
100.202
